ui/app/services/flags.ts

```diff
@@ -38,7 +38,10 @@ export default class flagsService extends Service {
     return null;
   }
 
-  // TODO getter will be used in the upcoming persona service
+  get isManaged() {
+    return this.managedNamespaceRoot !== null;
```

Comment:
to futureproof this code i'd recommend swapping the logic here i.e. 

```typescript
get isManaged(): boolean {
   return this.flags && this.flags.includes(FLAGS.vaultCloudNamespace);
}

get managedNamespaceRoot(): string | null {
  return  this.isManaged ? 'admin' : null;
}
```

^ this makes `isManaged` more about checking the thing that actually determines if the cluster is managed or not, while `managedNamespaceRoot` can stay focused on answering "what is the name of the `root` namespace for this managed cluster?"

Reply:
smart!